### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by FrancesIgwe**
Adding comments by players who has scanned  a particular qr code has been implemented!

### CODE REVIEW

File: app/src/main/java/com/cmput301w23t09/qrhunter/qrcode/AddQRCodeFragment.java [hunk 1]
```diff
@@ -106,16 +111,6 @@ private void onAddQRClicked(View view) {
...
-
-                          // Add the player to the QR
-                          QRCodeDatabase.getInstance()
-                              .addPlayerToQR(
-                                  activePlayer,
-                                  qrCode,
-                                  ignored -> {
-                                    loadingButton.setVisibility(View.GONE);
-                                    this.dismiss();
-                                  });
```
Comment (jmmabanta) [inline app/src/main/java/com/cmput301w23t09/qrhunter/qrcode/AddQRCodeFragment.java:118]: Why was this removed?

Comment (WillQizza) [reply app/src/main/java/com/cmput301w23t09/qrhunter/qrcode/AddQRCodeFragment.java:118]: My bad there, was merging on her branch and I had thought it was useless during my merging. will fix.

**Review by WillQizza — DISMISSED** [review]

**Review by WillQizza — APPROVED** [review]

**Review by IreneRSun — APPROVED** [review]
